libs/blocks/cai/cai.js

```diff
@@ -0,0 +1,110 @@
+import { createTag, getConfig, loadStyle } from '../../utils/utils.js';
+import { createC2pa, selectFormattedGenerator, selectGenerativeInfo } from '../../deps/cai-tools.min.js';
+
+const miloLibs = getConfig().miloLibs ?? '/libs';
+loadStyle(`${miloLibs}/blocks/cai/cai.css`);
+
+const tooltipContent = ({ issuer, date, info, app, aiTool }) => `
+      <div class="title">
+        <h3>Content Credentials</h3>
+        <p id="cai-subtitle">Issued by ${issuer} on ${date}</p>
+      </div>
+        <hr />
+      <div class="content">
+        <p>${info}</p>
+        <hr />
+        <p><b>App or device used </b> ${app}</p>
+        <hr />
+        <p><b>AI tool used </b> ${aiTool}</p>
+      </div>
+  `;
+
+const loader = '<div class="loader"></div>';
+
+const c2paPromise = createC2pa({
+  wasmSrc:
+      `${miloLibs}/deps/cai-toolkit.wasm`,
+  workerSrc:
+      `${miloLibs}/deps/cai-worker.min.js`,
+});
+
+const extractMetadata = async (img) => {
+  const generativeInfoTypes = {
+    compositeWithTrainedAlgorithmicMedia: 'This image has been augmented, corrected or enhanced using a Generative AI model, such as with inpainting or outpainting operations',
+    trainedAlgorithmicMedia: 'This image has been created algorithmically using an Artificial Intellgence model trained on captured content',
+    default: 'An AI tool was not used in creating this image',
```

Comment:
I think even if it's US only for now, these should be placeholders. If the wording needs to change, it's an authoring task rather than a DEV task.

Reply:
Yes I'll turn these into placeholders.